numba/core/config.py

```diff
@@ -424,7 +424,7 @@ def avx_default():
 
         CAPTURED_ERRORS = _readenv("NUMBA_CAPTURED_ERRORS",
                                    _validate_captured_errors_style,
-                                   'old_style')
+                                   'default')
```

Comment:
This confuses the functions `use_new_style_errors()` and `use_old_style_errors()`, which only recognise `"new_style"` and `"old_style"` respectively, so neither returns `True` now.